### ISSUE TITLE
learning rate meegeven lukt niet

### ISSUE THREAD

**Issue by TycoHAN** [TycoHAN/MADS-ML-Tyco#1]
Hoi Raoul,

Het lukt mij niet om de learning rate aan te passen. Hij blijft aangeven dat het adam object niet callable is, maar ik kom er niet uit. Kun jij helpen? Alvast bedankt.. Gr, Tyco

model = DenseNeuralNetwork(num_classes=10, units1=64, units2=64, input_size=28*28)

settings = TrainerSettings(
    epochs=2,
    metrics= [metrics.Accuracy()],
    logdir="modellogs",
    train_steps= 50,
    valid_steps= 50,
    reporttypes=[ReportTypes.TENSORBOARD, ReportTypes.TOML],
)

trainer = Trainer(
    model=model,
    settings=settings,
    loss_fn=loss_fn,
    optimizer=torch.optim.Adam(model.parameters(),lr=0.01),
    traindataloader=trainstreamer,
    validdataloader=validstreamer,
    scheduler=torch.optim.lr_scheduler.ReduceLROnPlateau
)
trainer.loop()
